### DISCUSSION TITLE
Microsoft.Compute/virtualMachines resourceType not being pulled

### DISCUSSION THREAD

**Discussion by buisz** [Azure/AzOps#566] [Q&A]
Dear All,

AzOps is a wonderfull piece of code and I love using it.

I just came across a single resourceType which is Microsoft.Compute/virtualMachines which is not being pulled. Is there a reason for this?

**Comment by daltondhcp** [comment]
Hi,

Thank you for the question! 
First, to confirm, have you set `"Core.SkipResource":` to `false` in settings.json (setting 13 in https://github.com/azure/azops/wiki/settings)
If so, are other resources pulled properly?

**Reply by buisz** [reply]
Hello,

I was offline for some days, I did put the setting to false.

Is it okay if I validate which resource types cannot be pulled with AzOps from my own environment?

Some details:
- I have created a AzOps set-up in Azure DevOps with the instructions provided in the WIKI.
- I have created a Service Principal with Access Permissions in the Tenant Root Group.
- I am using the multienv set-up. This is important for us due to multi-customer environments.

If you can reply to me if this is okay or have other suggestions on what I possibly can do to verify or test?
